src/main/java/mate/academy/spring/validation/email/EmailValidator.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.spring.validation.email;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
```

Comment:
![image](https://user-images.githubusercontent.com/58442482/136769335-113f94b8-dfbb-4869-9da6-73e8cf64cf1c.png)
we don't need this in this PR

Reply:
fixed